docs/website/docs/index.md

```diff
@@ -186,8 +186,10 @@ using various languages.
 * :fontawesome-brands-discord:
   [IREE Discord server](https://discord.gg/wEWh6Z9nMU): Daily development
   discussions with the core team and collaborators
-* :fontawesome-solid-users: [iree-discuss email list](https://groups.google.com/forum/#!forum/iree-discuss):
-  Announcements, general and low-priority discussion
+* :fontawesome-solid-users: [iree-announce email list](https://lists.lfaidata.foundation/g/iree-announce):
+  Announcements
+* :fontawesome-solid-users: [iree-technical-discussion email list](https://lists.lfaidata.foundation/g/iree-technical-discussion):
+  General and low-priority discussion
```

Comment:
Could use `:fontawesome-solid-bullhorn:` for the announce icon

(Other icons: https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/)

Before:
![image](https://github.com/user-attachments/assets/a33acff6-05a5-42eb-a837-e508f2013903)

After:
![image](https://github.com/user-attachments/assets/875bc9a9-20ae-4a55-ba28-b1cd830dafb9)

Either is fine with me. A different icon may be too visually noisy.

edit: actually, should probably match the footer. Maybe use `envelope` for both?
![image](https://github.com/user-attachments/assets/ff198cb4-ef2f-4b81-b62a-356319e2f764)

Reply:
What about the following variant which also adds the community calendar:

![image](https://github.com/user-attachments/assets/d5b1a967-f04e-4b7e-a00f-77f4273c54c3)

If we want to reduce the noise, we can switch to use the envelope for both mailing lists though.

Reply:
We'll want to link to meeting recordings somewhere too, so having that in its own category makes sense.

Reply:
Yes, as I realize this, I will add this in a separate PR (as outlined above), mainly because I am currently not creative enough for a good commit message that combines both.